src/utils_format_graphite_test.c

```diff
@@ -81,6 +81,11 @@ DEF_TEST(metric_name) {
           .type_instance = "test: \"hello\"",
           .want_name = "example@com.test-foo@@test@.single-test@@@hello@",
       },
+      {
+          .plugin_instance = "foo (test)",
+          .type_instance = "test: \"hello\"",
+          .want_name = "example,com.test-foo,,test,.single-test,,,hello,",
```

Comment:
This testcase is currently failing now with message:
_not ok 12 - got = "example@com.test-foo@@test@.single-test@@@hello@ 42 1480063672
", want "example,com.test-foo,,test,.single-test,,,hello, 42 1480063672
"._


I think this should be rewritten to something like this to actually test comma handling:
```
      /* test escaping comma */
      {
          .plugin_instance = "foo (test)",
          .type_instance = "test,123,",
          .want_name = "example@com.test-foo@@test@.single-test@123@",
      },
```

Reply:
Thanks for help!